require-dbt-version docs reflect post-1.0 advice (#1017)
fishtown-analytics/docs.getdbt.com: website/docs/reference/project-configs/require-dbt-version.md

:::info YAML Quoting

This configuration needs to be interpolated by the yaml parser as a string. As such, you should quote the value of the configuration, taking care to avoid whitespace. For example:
```yml
# ✅ These will work
require-dbt-version: ">=1.0.0" # Double quotes are OK
require-dbt-version: '>=1.0.0' # So are single quotes

# ❌ These will not work
require-dbt-version: >=1.0.0 # No quotes? No good
require-dbt-version: ">= 1.0.0" # Don't put whitespace after the equality signs
```

:::

## Examples

### Specify a minimum dbt version

> Use a `>=` operator for a minimum boundary. In this example, this project will run with any version of dbt greater than or equal to 1.0.0.

Reviewer: ```suggestion
Use a `>=` operator for a minimum boundary. In the following example, this project will run with any version of dbt greater than or equal to 1.0.0.
```

<File name='dbt_project.yml'>

```yml
require-dbt-version: ">=1.0.0"
```

</File>

### Pin to a range

> Use a comma separated list for an upper and lower bound. In this example, this project will run with dbt 1.x.x.

Reviewer: ```suggestion
Use a comma separated list for an upper and lower bound. In the following example, this project will run with dbt 1.x.x.
```

<File name='dbt_project.yml'>

```yml
require-dbt-version: [">=1.0.0", "<2.0.0"]
```

</File>

OR

<File name='dbt_project.yml'>

```yml
require-dbt-version: ">=1.0.0,<2.0.0"
```

</File>

  
### Require a specific dbt version
:::caution Not recommended
With the release of major version 1.0 of dbt Core, pinning to a specific patch is discouraged.
:::

> Use an exact version number. In this example, this project will only run with dbt v0.21.1. 

Reviewer: ```suggestion
While you can restrict your project to run only with an exact version of dbt Core, we do not recommend this for dbt Core v1.0.0 and higher. 

In the following example, the project will only run with dbt v0.21.1. 
```
